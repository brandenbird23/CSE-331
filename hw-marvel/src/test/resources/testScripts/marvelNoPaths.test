# Tests a pair of characters where no path exists

LoadGraph marvel marvel.csv

FindPath marvel SACRIFICER HOFFMAN