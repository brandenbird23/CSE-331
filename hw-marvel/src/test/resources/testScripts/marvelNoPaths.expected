# Tests a pair of characters where no path exists

loaded graph marvel

path from SACRIFICER to HOFFMAN:
no path found